tests/test_interpret.py

```diff
@@ -0,0 +1,224 @@
+    data = bmb.load_data("sleepstudy")
+    model = bmb.Model("Reaction ~ 1 + Days + (Days | Subject)", data)
+    idata = model.fit(tune=500, draws=500, random_seed=1234)
+    return model, idata
+
+
+@pytest.fixture(scope="module")
+def food_choice():
+    """
+    Model a categorical response using the 'categorical' family to test 'interpret'
+    plotting functions for a model whose predictions have multiple response
+    dimensions (levels).
+    """
+    length = [
+        1.3,
+        1.32,
+        1.32,
+        1.4,
+        1.42,
+        1.42,
+        1.47,
+        1.47,
+        1.5,
+        1.52,
+        1.63,
+        1.65,
+        1.65,
+        1.65,
+        1.65,
+        1.68,
+        1.7,
+        1.73,
+        1.78,
+        1.78,
+        1.8,
+        1.85,
+        1.93,
+        1.93,
+        1.98,
+        2.03,
+        2.03,
+        2.31,
+        2.36,
+        2.46,
+        3.25,
+        3.28,
+        3.33,
+        3.56,
+        3.58,
+        3.66,
+        3.68,
+        3.71,
+        3.89,
+        1.24,
+        1.3,
+        1.45,
+        1.45,
+        1.55,
+        1.6,
+        1.6,
+        1.65,
+        1.78,
+        1.78,
+        1.8,
+        1.88,
+        2.16,
+        2.26,
+        2.31,
+        2.36,
+        2.39,
+        2.41,
+        2.44,
+        2.56,
+        2.67,
+        2.72,
+        2.79,
+        2.84,
+    ]
+    choice = [
+        "I",
+        "F",
+        "F",
+        "F",
+        "I",
+        "F",
+        "I",
+        "F",
+        "I",
+        "I",
+        "I",
+        "O",
+        "O",
+        "I",
+        "F",
+        "F",
+        "I",
+        "O",
+        "F",
+        "O",
+        "F",
+        "F",
+        "I",
+        "F",
+        "I",
+        "F",
+        "F",
+        "F",
+        "F",
+        "F",
+        "O",
+        "O",
+        "F",
+        "F",
+        "F",
+        "F",
+        "O",
+        "F",
+        "F",
+        "I",
+        "I",
+        "I",
+        "O",
+        "I",
+        "I",
+        "I",
+        "F",
+        "I",
+        "O",
+        "I",
+        "I",
+        "F",
+        "F",
+        "F",
+        "F",
+        "F",
+        "F",
+        "F",
+        "O",
+        "F",
+        "I",
+        "F",
+        "F",
+    ]
+    sex = ["Male"] * 32 + ["Female"] * 31
+    data = pd.DataFrame({"choice": choice, "length": length, "sex": sex})
+    data["choice"] = pd.Categorical(
+        data["choice"].map({"I": "Invertebrates", "F": "Fish", "O": "Other"}),
+        ["Other", "Invertebrates", "Fish"],
+        ordered=True,
+    )
+
+    model = bmb.Model("choice ~ length + sex", data, family="categorical")
+    idata = model.fit(tune=500, draws=500, random_seed=1234)
+    return model, idata
+
+
+@pytest.mark.parametrize("return_idata", [True, False])
+def test_return_idata_common_effects(mtcars, return_idata):
+    model, idata = mtcars
+
+    bmb.interpret.predictions(
```

Comment:
Can we add a check for the type of the object returned and the number of rows/columns? Notice we would need to fix the number of chains above.